presto-main/src/main/java/io/prestosql/sql/analyzer/StatementAnalyzer.java

```diff
@@ -2002,7 +1997,7 @@ private RelationType analyzeView(Query query, QualifiedObjectName name, Optional
             }
             catch (RuntimeException e) {
                 throwIfInstanceOf(e, PrestoException.class);
-                throw new SemanticException(VIEW_ANALYSIS_ERROR, node, "Failed analyzing stored view '%s': %s", name, e.getMessage());
+                throw semanticException(VIEW_ANALYSIS_ERROR, node, "Failed analyzing stored view '%s': %s", name, e.getMessage());
```

Comment:
Hmm.. I realized I'll need to adjust this `catch` block, too, since it rethrows PrestoException. Now every semantic error will be a PrestoException, so it will almost never get to this line.